data/toc.yaml

```diff
@@ -563,12 +563,24 @@ Reference:
       section:
       - path: /engine/reference/commandline/scout/
         title: docker scout
+      - path: /engine/reference/commandline/scout_cache/
+        title: docker scout cache
+      - path: /engine/reference/commandline/scout_cache_df/
```

Comment:
Some of the other subcommands are nested under the parent. Not sure if it applies here for cache, just noting.